cl/users/email_handlers.py

```diff
@@ -676,3 +677,83 @@ def enqueue_email(recipients: list[str], message_id: str) -> None:
             f"The message: {message_id} can't be enqueued because it "
             "doesn't exist anymore."
         )
+
+
+PERMANENT = {
+    EMAIL_NOTIFICATIONS.GENERAL: "Your email provider sent a hard "
+    "bounce message indicating we can't continue sending messages to your "
+    "current email address. "
+    "Please consider changing your email address.",
+    EMAIL_NOTIFICATIONS.NO_EMAIL: "Your email provider sent a bounce "
+    "message indicating that your email address doesn't exist. Please "
+    "consider changing your email address.",
+    EMAIL_NOTIFICATIONS.COMPLAINT: "We've received a complaint from "
+    "your email provider indicating that you don't want to receive emails "
+    "from us. Please consider changing your email address.",
+}
+
+TRANSIENT = {
+    EMAIL_NOTIFICATIONS.GENERAL: "Your email provider sent a bounce message. "
+    "We are still attempting to fix this problem by resending "
+    "messages.",
+    EMAIL_NOTIFICATIONS.MAILBOX_FULL: "Your email provider sent a "
+    "bounce message because your inbox was full. Please consider clearing out "
+    "your email inbox.",
+}
+
+
+def determine_email_error_message(type: str, subtype: int) -> str:
+    """Determine the message to display to the user with more details
+    about the Permanent or Transient email address error.
+
+    :param type: The type of email address error, Permanent or Transient.
+    :param subtype: The error subtype returned by the SES notification.
```

Comment:
Can we update these parameters to be more descriptive?  

I dislike `type` as a variable.  It so close to a reserved name.

Reply:
I deleted this function as part of the simplification.